### ISSUE TITLE
Exception when adding an image to product or when saving product that has image

### ISSUE THREAD

**Issue by merzilla** [aimeos/ai-admin-jqadm#45]
Hi,

with TYPO3 8.7.16 I get an exception when either saying a product that already has an image, or if I delete the image and "save" it works, but adding an image again leads to the following exception:
```
Invalid size provided for UploadedFile; must be an int 

InvalidArgumentException thrown in file
/Users/path/to/my/site/www/vendor/zendframework/zend-diactoros/src/UploadedFile.php in line 114.
```

I don't know if this is an Aimeos issue or if the TYPO3 core has a buggy version of Zend dependencies.
Can you reproduce this?

Thanks a lot and regards

**Comment by aimeos** [comment]
This has been fixed in dev-master and will be part of the 2018.07 release this week

**Comment by merzilla** [comment]
Oh, great news! Thank you very much.
